Profile: Update

CongHuanPTIT opened the pull request:
- Adding fragments
- Identifying bundle() in Profile Edit Fragment 
- Connecting screens in nav_graph: 2-way between Profile and ProfileEdit

Also added comments in places that needs review/knowledge.

!!! Accidentally pushed with Common.java changed to my URL. Change back if possible. Thank you

CongHuanPTIT committed:
Adding fragment files: ProfileFragment.java + ProfileEditFragment.java
Adding profile fragment XML file into nav_graph.xml

CongHuanPTIT committed:
Create base for profile view and update: request + response file, connecting the screens in nav_graph.xml, service

CongHuanPTIT committed:
- Update ProfileService.java
- Change fragment_profile.xml to ScrollView to include Order Status (RecyclerView, placeholder for now)
- Change UI for the XML profile files
- Request + Response files needs reviewing

huypham85 commented on app/src/main/java/com/example/e_commerce/network/model/request/CurrrentUserRequest.java:
```diff
@@ -0,0 +1,44 @@
+package com.example.e_commerce.network.model.request;
+
+public class CurrrentUserRequest {
```
file Request này dùng cho api nào vậy?

CongHuanPTIT replied:
T định là để Request và Response cho API Post (user/update)

CongHuanPTIT replied:
T định là để Request và Response cho API Post (user/update)

huypham85 commented on app/src/main/java/com/example/e_commerce/network/model/response/CurrentUserResponse.java:
```diff
@@ -0,0 +1,5 @@
+package com.example.e_commerce.network.model.response;
+
+public class CurrentUserResponse {
```
bên dưới t thấy có 1 file CurrentUserResponse rồi, nếu k dùng file này thì xoá đi

CongHuanPTIT committed:
wip: Resolving problem at ProfileFragment.java

CongHuanPTIT committed:
wip:
- Working on profile fragment files
- Needs to add User details and put in API
- Add order_status.xml

CongHuanPTIT committed:
Merge remote-tracking branch 'origin/main' into profile

# Conflicts:
#	app/src/main/java/com/example/e_commerce/di/ServiceModule.java
#	app/src/main/java/com/example/e_commerce/utils/Common.java
#	app/src/main/res/navigation/nav_graph.xml

CongHuanPTIT committed:
wip:
- Working on profile fragment files
- Needs to add User details and put in API
- Add order_status.xml
